src/encoded/commands/deploy.py

```diff
@@ -1009,7 +1009,7 @@ def hostname(value):
     )
     parser.add_argument('--es-wait', action='store_true', help="Create es nodes and head node.")
     parser.add_argument('--cluster-name', default=None, type=hostname, help="Name of the cluster")
-    parser.add_argument('--cluster-size', type=int, default=5, help="Elasticsearch cluster size")
+    parser.add_argument('--cluster-size', default=5, type=int, help="Elasticsearch cluster size")
```

Comment:
the changes in this file can all be reverted.  The type=int is already in origin/dev.  The other lines are not needed since the parser sets the arg as an int.